spec/integration/client_spec.rb

```diff
@@ -1,8 +1,5 @@
 require "spec_helper"
 
-java_import java.util.logging.Logger
-java_import java.util.logging.Level
```

Comment:
Maybe we should leave Level here, since we use it below. Suppose it's implicity added by the `spec_helper` now